pkgs/top-level/all-packages.nix

```diff
@@ -2080,6 +2083,10 @@ with pkgs;
     stdenv = gcc15Stdenv;
   };
 
+  hyprlauncher = callPackage ../by-name/hy/hyprlauncher/package.nix {
+    stdenv = gcc15Stdenv;
+  };
```

Comment:
Please have a look at https://github.com/NixOS/nixpkgs/pull/453948/commits/1aafd741004439f420409157e7f2f41182668bc0 and don't introduce this pattern anymore - we are going to assert against it very soon in https://github.com/NixOS/nixpkgs/pull/454525.